test/simulator/abilities/normalize.js

```diff
@@ -30,7 +30,7 @@ describe('Normalize', function () {
 		battle = common.createBattle();
 		battle.join('p1', 'Guest 1', 1, [{species: "Delcatty", ability: 'normalize', item: 'dousedrive', moves: ['technoblast']}]);
 		battle.join('p2', 'Guest 2', 1, [{species: "Latias", ability: 'colorchange', moves: ['endure']}]);
-		battle.makeChoices('move technoblash', 'move endure');
+		battle.makeChoices('move technoblast', 'move endure');
```

Comment:
Heh, maybe choice errors should just throw in tests.

Reply:
Yeah, I was discovering that yesterday as well - I think the fact that its so magically lenient can be troublesome. Throwing in tests seems reasonable to me.